Simplified/Reader2/UI/NYPLBaseReaderViewController.swift

```diff
@@ -364,6 +344,13 @@ extension NYPLBaseReaderViewController: NavigatorDelegate {
         return nil
       }
     }()
+    
+    if let resourceIndex = publication.resourceIndex(forLocator: locator),
```

Comment:
just curious did you find this with unit tests?

Reply:
No, the bookmark button wasn't updating when I navigate between the pages, so I just fixed it.